azkaban-exec-server/src/main/java/azkaban/execapp/FlowRampManager.java

```diff
@@ -629,8 +629,8 @@ synchronized private Action convertToAction(Status status) {
 
   private static class RampDataModel {
     // Host the current processing ramp flows
-    // Map.Key = flowName, Map.Value = Set Of Ramps
```

Comment:
I think this comment does get impacted by the changes. It will help to modify this as well